sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/SyncActivity.kt

```diff
@@ -272,9 +282,14 @@ class SyncActivity : DuckDuckGoActivity() {
                 finish()
             }
             is RequestSetupAuthentication -> launchDeviceAuthEnrollment()
+            is LaunchSyncGetOnOtherPlatforms -> launchSyncGetOnOtherPlatforms()
         }
     }
 
+    private fun launchSyncGetOnOtherPlatforms() {
+        startActivity(SyncGetOnOtherPlatformsActivity.intent(this))
```

Comment:
Why not use GlobalActivityStarter?

Reply:
👍. This will be done in the upper branch: https://github.com/duckduckgo/Android/pull/4921